### PR TITLE
Cp2k fix

### PR DESCRIPTION
**Pull request by lucamar**
Adding CP2K-trunk-CrayGNU-2016.11-cuda-8.0.54 to fix a bug in CP2K release 4.1: the fix is available only in the trunk SVN, therefore I needed to package the source tarball separately.

### CODE REVIEW

**Comment by lucamar** [comment]
Successful after shortening the buildpath of user jenscscs! Thanks @gppezzi

**Comment by gppezzi** [comment]
Good but I suggest you add a date/tag to the version (if that does not break the build 😄 )

File: easybuild/easyconfigs/c/CP2K/CP2K-svn-CrayGNU-2016.11-cuda-8.0.54.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'MakeCp'
+
+name = 'CP2K'
+version = "svn"
```
Comment (gppezzi) [inline easybuild/easyconfigs/c/CP2K/CP2K-svn-CrayGNU-2016.11-cuda-8.0.54.eb]: how about:
version = "2017.02"? 

Unless you don't plan to make it public (in that case you can leave it like this).

Comment (lucamar) [inline easybuild/easyconfigs/c/CP2K/CP2K-svn-CrayGNU-2016.11-cuda-8.0.54.eb]: Right now I was not planning to build it with Jenkins for the general public in production (it is not a stable official release), but just send it to two users to fix their bug. I hope the CP2K developers will soon publish a new stable release with the bug fix.

**Pull request merged**